RELEASE/scripts/autoscend/auto_bedtime.ash

```diff
@@ -447,6 +452,20 @@ void bedtime_pulls()
 		return;
 	}
 	
+	if(get_property("auto_bedtime_pulls_min_desirability").to_float() <= 5.0)
+	{
+		if(item_amount($item[potato alarm clock]) == 0 && storage_amount($item[potato alarm clock]) > 0)
+		{
+			pullXWhenHaveY($item[potato alarm clock], 1, 0);
+		}
```

Comment:
```suggestion
		pullXWhenHaveY($item[potato alarm clock], 1, 0);
```
the if statement is wholly redundant with `pullXWhenHaveY`
as `pullXWhenHaveY(it, 1, 0)` already does this if statement

also, I don't think this pull should be inside the
`if(get_property("auto_bedtime_pulls_min_desirability").to_float() <= 5.0)` but come before it.